pkg/cli/containers_rm.go

```diff
@@ -0,0 +1,53 @@
+package cli
+
+import (
+	"fmt"
+
+	cli "github.com/acorn-io/acorn/pkg/cli/builder"
+	"github.com/acorn-io/acorn/pkg/client"
+	"github.com/spf13/cobra"
+)
+
+func NewContainerDelete() *cobra.Command {
+	cmd := cli.Command(&ContainerDelete{}, cobra.Command{
+		Use: "rm [CONTAINER_NAME...]",
```

Comment:
per slack discussion, lets change this to kill. can you add a little more to the description to explain that the input to this should be the output from the acorn containers command? if it gets to wordy to explain in cli text, dont worry about it